torch/autograd/ir.py

```diff
@@ -0,0 +1,4 @@
+import torch._C as _C
+
+class Node(_C._NodeBase):
```

Comment:
So I think `Node` is not going to be a good name for the thing. When talking about autograd, we often say 'node' when we think about Functions, and this is going to make it quite ambiguous. Perhaps `IRNode` would be better?